docs/resources/lts_transfer.md

```diff
@@ -0,0 +1,311 @@
+
+* `agency_project_id` - (Required, String, ForceNew) Project ID of the delegator.
+
+  Changing this parameter will create a new resource.
+
+<a name="LtsTransfer_LogTransferDetail"></a>
+The `log_transfer_detail` block supports:
+
+* `obs_period` - (Optional, Int) Length of the transfer interval for an OBS transfer task.  
+  This parameter is mandatory when you create an OBS transfer task.  
+  The log transfer interval is specified by the combination of the values of **obs_period** and **obs_period_unit**,
+  and must be set to one of the following: 2 min, 5 min, 30 min, 1 hour, 3 hours, 6 hours, and 12 hours.
+  Value options are as follows:
+    + **2**: 2 minutes, the **obs_period_unit** must be **min**.
+    + **5**: 5 minutes, the **obs_period_unit** must be **min**.
+    + **30**: 30 minutes, the **obs_period_unit** must be **min**.
+    + **1**: 1 hour, the **obs_period_unit** must be **hour**.
+    + **3**: 3 hours, the **obs_period_unit** must be **hour**.
+    + **6**: 6 hours, the **obs_period_unit** must be **hour**.
+    + **12**: 12 hours, the **obs_period_unit** must be **hour**.
+
+* `obs_period_unit` - (Optional, String) Unit of the transfer interval for an OBS transfer task.  
+  This parameter is mandatory when you create an OBS transfer task.
+  The log transfer interval is specified by the combination of the values of **obs_period** and **obs_period_unit**,
+  and must be set to one of the following: 2 min, 5 min, 30 min, 1 hour, 3 hours, 6 hours, and 12 hours.
+  Value options are as follows:
+    + **min**: minute.
+    + **hour**: hour.
+
+* `obs_bucket_name` - (Optional, String) OBS bucket name.  
+  This parameter is mandatory when you create an OBS transfer task.
+
+* `obs_transfer_path` - (Optional, String) OBS bucket path, which is the log transfer destination.  
+
+* `obs_dir_prefix_name` - (Optional, String) Custom transfer path of an OBS transfer task.  
+
+* `obs_prefix_name` - (Optional, String) Transfer file prefix of an OBS transfer task.  
+
+* `obs_eps_id` - (Optional, String) Enterprise project ID of an OBS transfer task.  
+
+* `obs_encrypted_enable` - (Optional, Bool) Whether OBS bucket encryption is enabled.  
+
+* `obs_encrypted_id` - (Optional, String) KMS key ID for an OBS transfer task.  
+  This parameter is mandatory if encryption is enabled for the target OBS bucket.  
+
+* `obs_time_zone` - (Optional, String) Time zone for an OBS transfer task.  
+  For details, see [Time Zone List for OBS Transfer](https://support.huaweicloud.com/intl/en-us/api-lts/lts_api_0111.html).
+  If this parameter is specified, **obs_time_zone_id** must also be specified.
+
+* `obs_time_zone_id` - (Optional, String) ID of the time zone for an OBS transfer task.  
+  For details, see [Time Zone List for OBS Transfer](https://support.huaweicloud.com/intl/en-us/api-lts/lts_api_0111.html).
+  If this parameter is specified, **obs_time_zone** must also be specified.
+
```

Comment:
How about add descriptions for these fields in log_transfer_detail, Explain the relationship between fields and type `log_transfer_type`.

  ->The fields `obs_period`, `obs_period_unit`, `obs_bucket_name`, `obs_transfer_path`, `obs_dir_prefix_name`, `obs_prefix_name`, `obs_eps_id`, `obs_encrypted_enable`, `obs_encrypted_id`, `obs_time_zone` and `obs_time_zone_id` only valid when `log_transfer_type` is set to **OBS**.